src/ray/rpc/grpc_server.cc

```diff
@@ -135,17 +135,22 @@ void GrpcServer::Run() {
 
   // Create calls for all the server call factories.
   for (auto &entry : server_call_factories_) {
-    for (int i = 0; i < num_threads_; i++) {
-      // Create a buffer of 100 calls for each RPC handler.
-      // TODO(edoakes): a small buffer should be fine and seems to have better
-      // performance, but we don't currently handle backpressure on the client.
-      int buffer_size = 100;
-      if (entry->GetMaxActiveRPCs() != -1) {
-        buffer_size = entry->GetMaxActiveRPCs();
-      }
-      for (int j = 0; j < std::max(1, buffer_size / num_threads_); j++) {
-        entry->CreateCall();
-      }
+    // Derive target max inflight RPCs buffer based on `gcs_max_active_rpcs_per_handler`
+    //
+    // NOTE: For these handlers that have set it to -1, we set default (per
+    //       thread) buffer at 32, though it doesn't have any impact on concurrency
+    //       (since we're recreating new instance of `ServerCall` as soon as one
+    //       gets occupied therefore not serving as back-pressure mechanism)
+    int buffer_size;
+    if (entry->GetMaxActiveRPCs() != -1) {
+      buffer_size = std::max(1, int(entry->GetMaxActiveRPCs() / num_threads_));
+    } else {
+      buffer_size = 32;
```

Comment:
why 100 -> 32?

Reply:
It was 100 for all threads, i'm creating 32 per thread now